GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/Utilities/ProcessRunner.m

```diff
@@ -0,0 +1,59 @@
+#import "ProcessRunner.h"
+
+@interface ProcessRunner()
+
+@property (strong) ProcessFactory processFactory;
+
+@end
+
+@implementation ProcessRunner
+
+- (id) initWithProcessFactory:(ProcessFactory) processFactory
+{
+    if (self = [super init])
+    {
+        self.processFactory = processFactory;
+    }
+    
+    return self;
+}
+
+- (BOOL) tryRunExecutable:(NSURL *) path
+    args:(NSArray *) args
+    output:(NSString **) output
+    error:(NSError **) error
+{
+    NSTask *task = self.processFactory();
+    NSPipe *taskOut = [NSPipe pipe];
+    
+    task.executableURL = path;
+    task.arguments = args;
+    task.standardOutput = taskOut;
+    task.standardError = taskOut;
+    
+    int exitCode = -1;
+    
+    if ([task launchAndReturnError:error])
+    {
+        [task waitUntilExit];
+        
+        exitCode = [task terminationStatus];
+        
+        *output = [[NSString alloc]
+            initWithData:[taskOut.fileHandleForReading availableData]
+            encoding:NSUTF8StringEncoding];
+        
+        if (0 != exitCode)
+        {
+            NSDictionary *userInfo = @{ NSLocalizedDescriptionKey : *output };
+            *error = [NSError errorWithDomain:@"ProcessRunner"
+                code:exitCode
```

Comment:
Curious, what IDE/text editor are you using? Usually xcode aligns the ":" in the method parameter list when spread across multiple lines. If you use clang-format, that will also format the file appropriately for ObjC.

Reply:
Using Xcode. I was following indentations used in our C# code. Restored the default Xcode indentations now.